docs/cubeviz/export_data.rst

```diff
@@ -15,6 +15,35 @@ Spatial Regions
     :ref:`Export Spatial Regions <imviz_export>`
         Documentation on how to export spatial regions.
 
+Since Specviz can be accessed from Cubeviz, the following line of code
+can be used to extract the spectrum of a spatial subset named "Subset 1"::
+
+    subset1_spec1d = cubeviz.specviz.get_spectra("Subset 1")
+
+An example without accessing Specviz::
+
+    subset1_spec1d = cubeviz.app.get_data_from_viewer("flux-viewer", data_label="Subset 1")
+
+To get all subsets from a viewer::
+
+    subset1_spec1d = cubeviz.app.get_subsets_from_viewer("spectrum-viewer")
+
+To access the spatial region itself::
+
+    dc = cubeviz.app.data_collection
+    subset1 = dc.subset_groups[0]
+    x, y = subset1.subset_state.roi.get_center()
+    radius = subset1.subset_state.roi.radius
+
+The syntax to output a subset as an astropy region looks like::
+
+    dc.subset_groups[0].subsets[0].data.get_selection_definition(subset_id="Subset 1", format='astropy-regions')  # noqa
+
+Where the first `0` corresponds to what number subset it is and the second `0` corresponds
+to which data the subset is being applied to (flux, variance, data quality, etc.).
+If you have multiple subsets present in the application, you need to fill out the
+`subset_id` parameter.
+
```

Comment:
How does this look @PatrickOgle?

Reply:
That's better, but what if it isn't a circle?

Reply:
Ok, added information on extracting rectangular ROIs.